sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -106,21 +110,25 @@
   private static Cluster cluster;
   private static Session session;
 
-  @ClassRule public static final TemporaryFolder TEMPORARY_FOLDER = new TemporaryFolder();
+  private static final String TEMPORARY_FOLDER = System.getProperty("java.io.tmpdir");
   @Rule public transient TestPipeline pipeline = TestPipeline.create();
   private static CassandraShutDownHook shutdownHook;
 
   @BeforeClass
   public static void beforeClass() throws Exception {
     jmxPort = NetworkTestHelper.getAvailableLocalPort();
     shutdownHook = new CassandraShutDownHook();
-    // randomized port at startup
-    String data = TEMPORARY_FOLDER.newFolder("embedded-cassandra", "data").getPath();
-    String commitLog = TEMPORARY_FOLDER.newFolder("embedded-cassandra", "commit-log").getPath();
-    String cdcRaw = TEMPORARY_FOLDER.newFolder("embedded-cassandra", "cdc-raw").getPath();
-    String hints = TEMPORARY_FOLDER.newFolder("embedded-cassandra", "hints").getPath();
-    String savedCache = TEMPORARY_FOLDER.newFolder("embedded-cassandra", "saved-cache").getPath();
-    cluster =
+    String data = TEMPORARY_FOLDER + "/embedded-cassandra/data";
```

Comment:
Please, extract `TEMPORARY_FOLDER + "/embedded-cassandra/"` into another const string since it appears in many other places in this test.

Reply:
done